src/i18n.js

```diff
@@ -17,12 +25,26 @@ i18n.use(initReactI18next).init({
   nsSeparator: '~',
   keySeparator: false,
   debug: false,
+  interpolation: {
+    format(value, format, lng) {
+      if (format === 'number') {
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#Browser_compatibility
+        return new Intl.NumberFormat(lng).format(value);
+      }
+      if (value instanceof Date) {
+        return dateTimeFormatter.format(value);
+      }
+      return value;
+    },
+    escapeValue: false, // not needed for react as it escapes by default
+  },
   react: {
     useSuspense: true,
     transSupportBasicHtmlNodes: true, // allow <br/> and simple html elements in translations
   },
-  missingKeyHandler: function (lng, ns, key) {
-    window.windowError = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`;
+  missingKeyHandler(lng, ns, key) {
+    window.windowError = `Missing i18n key '${key}' in namespace '${ns}' and language '${lng}.'`;
```

Comment:
why are we adding `windowError` to `window` ? Can't we just directly  do `console.error` ?